packages/astro/src/@types/astro.ts

```diff
@@ -1532,6 +1532,45 @@ export interface AstroUserConfig {
 			 * - `"pathanme": The strategy is applied to the pathname of the URLs
 			 */
 			strategy: 'pathname';
+
+			/**
+			 * @name experimental.i18n.domains
```

Comment:
```suggestion
			 * @name i18n.domains
```
Just confirming that this should (or shouldn't?) have experimental in the name, since it's not actually used/configured under an experimental flag?